src/input_parsers/NetworkParser.h

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
Update the header to match template like in other files and provide a class description.